lua/luasnip/nodes/snippet.lua

```diff
@@ -875,6 +1144,7 @@ function Snippet:del_marks()
 	end
 end
 
+-- FIXME: `info` is _never_ used anywhere?
```

Comment:
Oh, this is very hidden: `is_interactive` is overridden for a few ast-parsed nodes, and _that_ version uses `info` (or, more precisely, `env`)

Reply:
good catch, I tried to find usage with luals, but ripgrep found more :grimacing:

Reply:
Yeah, there are a few.. unorthodox things like that in the codebase :D